opapi/readme.md

```diff
@@ -92,3 +92,8 @@ The Botpress Engineering team.
 ## Licensing
 
 This software is protected by the same license as the [main Botpress repository](https://github.com/botpress/botpress). You can find the license file [here](https://github.com/botpress/botpress/blob/master/LICENSE).
+
+## Publishing
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏ You should add this in the README.md at the root of the repository. I don't think we want to add the publishing steps of a npm package inside the npm README.
![image](https://user-images.githubusercontent.com/10071388/234939975-32aa6fcb-0d9a-4bf0-8edd-43646d28c6e3.png)

Reply:
I added instructions to `opapi/CONTRIBUTING.md` since instructions may vary depending on package

Reply:
That's good but we should add it to the npm ignore `opapi/.npmignore`.

Reply:
@michaelmass done in 9e4dcc6